connect/service.go

```diff
@@ -26,13 +27,31 @@ type service struct {
 	commands command.Service
 }
 
-func (svc service) Acknowledge(deviceUDID, commandUUID string) (int, error) {
-	total, err := svc.commands.DeleteCommand(deviceUDID, commandUUID)
+func (svc service) Acknowledge(ctx context.Context, req mdmConnectRequest) (int, error) {
+	var err error
+
+	switch req.RequestType {
+	case "DeviceInformation":
+		err = svc.ackQueryResponses(req)
```

Comment:
Instead of declaring `var err error` at the top, I would instead write this as 
```
if err := svc.ackQueryResponses(req); err !=nil {
    return 0, err
}
```